packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2681,6 +2681,16 @@ class WidgetToRenderBoxAdapter extends LeafRenderObjectWidget {
   }
 }
 
+class SliverToBoxAdapter extends SingleChildRenderObjectWidget {
+  SliverToBoxAdapter({
+    Key key,
+    Widget child,
```

Comment:
It's a render object widget, and the render object can handle not having a child.